javascript/operations.js

```diff
@@ -254,6 +254,21 @@ export default {
     })
   },
 
+  setMeta: operation => {
+    dispatch(document, 'cable-ready:before-set-meta', operation)
+    const { name, content } = operation
+    if (!operation.cancel) {
+      let meta = document.head.querySelector(`meta[name=${name}]`)
```

Comment:
We'll likely want to quote `name` in the selector. 

```js
let meta = document.head.querySelector(`meta[name='${name}']`)
```

Reply:
Good catch, although it's interesting that it's worked without it.

Reply:
I think it doesn't make a difference.

Looking at:
https://stackoverflow.com/questions/5578845/css-attribute-selectors-the-rules-on-quotes-or-none

> You can usually omit the quotes as long as the attribute value is alphanumeric